keel-web/src/test/resources/examples/delivery-config-example.yml

```diff
@@ -113,10 +113,9 @@ environments:
     regions:
     - name: us-west-2
   resources:
-  - kind: ec2/cluster@v1
+  - kind: ec2/cluster@v1.1
     spec:
-      imageProvider:
-        reference: fnord
+      artifactReference: fnord
       moniker:
         app: fnord
         stack: staging
```

Comment:
Example of what the difference between the old & new cluster spec formats looks like

Reply:
ooo this is sparking lots of joy

Reply:
luv it!